base/docs/helpdb/Dates.jl

```diff
@@ -82,8 +128,9 @@ Dates.Date(dt::DateTime)
 """
     Date(dt::AbstractString, format::AbstractString; locale="english") -> Date
 
-Construct a `Date` type by parsing a `dt` date string following the pattern given in the
-`format` string. Follows the same conventions as `DateTime` above.
+Construct a `Date` object by parsing a `dt` date string following the pattern given in the
+`format` string. Follows the same conventions as
+`DateTime(::AbstractString, ::AbstractString)`.
```

Comment:
Ahh, I don't know you couldn't do that. I wonder whether the last one could be made to work (and maybe even the second one if the signature matches). @MichaelHatherly?

But for now keep it that way then.